pkg/lokifrontend/frontend/v2/frontend.go

```diff
@@ -259,13 +272,22 @@ func (f *Frontend) QueryResult(ctx context.Context, qrReq *frontendv2pb.QueryRes
 			level.Warn(f.log).Log("msg", "failed to write query result to the response channel", "queryID", qrReq.QueryID, "user", userID)
 		}
 	}
+	// TODO(chaudum): In case the the userIDs do not match, we do not send a
```

Comment:
I understand this TODO will be addressed on a separate PR?

Reply:
Right